packages/plugin-ext/src/plugin/env.ts

```diff
@@ -46,4 +47,8 @@ export class EnvExtImpl {
     setQueryParameters(queryParams: QueryParameters): void {
         this.queryParameters = queryParams;
     }
+
+    getOsType(): Promise<theia.OSType> {
```

Comment:
would be good to be explicit about whether this is the frontend or backend OS.

Reply:
Yes, I was wondering when adding it. (it's on proposed step so we can change :)

So you would prefer to add it inside the method name ?
like 
```typescript
getFrontendOsType()
```
or

```typescript
getClientOsType()
```

Reply:
Yes, I think that would help. On plug-in level 'client' might make sense, because it is about the concrete connection.

Reply:
sure, so `theia.env.getClientOsType()` or `theia.env.getClientOS()` method ?

or as a read-only attribute (I was wondering also when introducing it)

`theia.env.clientOSType`, `theia.env.clientOperatingSystem`, etc

Reply:
all seems fine :-)